doc/customization_basic.rst

```diff
@@ -0,0 +1,150 @@
+        download_osm_buildings: true  # If true, OpenStreetMap buildings will be downloaded for the selected countries
+        download_osm_method: overpass # or earth_osm
+        build_cutout: false
+        build_natura_raster: false  # If True, than an exclusion raster will be build
+        
+        # If "build_cutout" : true, then environmental data is extracted according to `snapshots` date range and `countries`
+          # prepare_links_p_nom: false
+
+
+Regarding topics in the enable section:
+
+- Retrive_databundle and retrive_cost_data: After the initial run, it is a good idea to set retrive databundle and cost data to false to avoid re-downloading unnecessary data.
+- Dowload_osm_data and download_osm_buildings: These two options are used to download OpenStreetMap data for the selected countries. The download_osm_method option allows you to choose the method to use to download the data. The overpass method is the default method, and currently the most effective.
+- Build_cutout: When using weather year 2013 it is advisable to set "build_cutout: false" because the precompiled cutouts are automatically downloaded with the "retrive_databundle: true". When using a weather year other than 2013 it is essential to set "build_cutout: true" to generate custom cutouts.
+    Caution: when using the Build_cutout rule, it is essential to first configure the Copernicus Climate Data Store API ( read the instructions).
+    After the first run and successful custom cutout generation, build_cutout can be switched to false to avoid rebuilding the cutout.
+- Build_nature_raster: When "build_nature_raster" is configured to false, the exclusion raster for protected areas is taken from the precompiled file "data/nature.tiff" downloaded with the databundle.Conversely, if "build_nature_raster" is set to "true" the exclusion raster is calculated using the "build_nature_raster" rule.
+  After the initial run, it is recommended to set the retrieval of databundle and cost data to ``false`` to prevent unnecessary redownloading of data.
+
+
+Specify the weather year scope
+------------------------------
+With these arguments, it is possible to set the time horizon of the simulation
+Likewise, the example's temporal scope can be restricted (e.g. to 7 days):
+
+.. code:: yaml
+
+    snapshots:
+        start: "2013-03-01"
+        end: "2013-03-07"
+        inclusive: "left" # end is not inclusive
+
+.. note::
+
+    Ensure that the selected date range aligns with the dates available in the cutout dataset. If the weather data within the cutouts corresponds to the year 2013, 
+    then the range of snapshots should fall within that same year.
+    In in the configuration file you will also find the variable "year" this specifies the reference year for the local population data. 
+    It is your choice whether or not to keep the data temporally concordant
+
+specify the method of load calculation :
```

Comment:
First letter uppercase and avoid the spacing at the end "calculation :" -> "calculation:"